pkg/apis/core/types.go

```diff
@@ -2022,6 +2022,14 @@ type Probe struct {
 	// Minimum consecutive failures for the probe to be considered failed after having succeeded.
 	// +optional
 	FailureThreshold int32
+	// Optional duration in seconds the pod needs to terminate gracefully upon probe failure.
+	// Value must be non-negative integer. The value zero indicates delete immediately.
```

Comment:
Don't use `delete` here.  Use `The value zero indicates stop immediately via the kill signal (no opportunity to shut down`.

Reply:
I copied this directly from the pod-level spec, should I update that comment as well?

Reply:
Yes, but we'll want this message to be slightly tweaked anyway (which i'll do on the public api).